packages/semantic-dom-diff/package.json

```diff
@@ -26,17 +26,15 @@
     "test:es5:watch": "karma start karma.es5.config.js --auto-watch=true --single-run=false",
     "prepublishOnly": "../../scripts/insert-header.js"
   },
-  "dependencies": {
-    "@bundled-es-modules/diffable-html": "^4.0.0-rc.2",
-    "@bundled-es-modules/parse5": "^5.1.0",
-    "@types/parse5": "^5.0.0"
-  },
   "devDependencies": {
     "@bundled-es-modules/chai": "^4.2.0",
     "@open-wc/testing-karma": "^0.4.15",
     "@open-wc/testing-karma-bs": "^0.2.17",
     "@open-wc/testing-wallaby": "^0.1.12",
     "mocha": "^5.0.0",
     "webpack-merge": "^4.1.5"
+  },
+  "dependencies": {
+    "htmlparser2": "^3.10.1"
```

Comment:
that will not work via polymer serve/owc-dev-server right?
so we should open an issue and make an estimate module version right?

Reply:
Indeed, it's a WIP item left to do. I was thinking of pre-bundling within `semantic-dom-diff` as a prepublish step. We can the minify it too to reduce the footprint when running tests. What do you think?